compression/peak_signal_to_noise_ratio.py

```diff
@@ -11,14 +11,16 @@
 import cv2
 import numpy as np
 
+PIXEL_MAX = 255.0
+
 
 def psnr(original: float, contrast: float) -> float:
     mse = np.mean((original - contrast) ** 2)
     if mse == 0:
         return 100
-    PIXEL_MAX = 255.0  # noqa: N806
-    PSNR = 20 * math.log10(PIXEL_MAX / math.sqrt(mse))  # noqa: N806
-    return PSNR
+
+    psnr = 20 * math.log10(PIXEL_MAX / math.sqrt(mse))
+    return psnr
```

Comment:
`flake8-return` would have suggested this and given that `psnr` is not at all self-documenting, I agree.
```suggestion
    return 20 * math.log10(PIXEL_MAX / math.sqrt(mse))
```